pizza-time/src/styles/eventStyles.js

```diff
@@ -18,38 +18,92 @@ export const Inner = styled.div`
   }
 `;
 
+export const Toggle = styled.span`
```

Comment:
Nice, you decided to implement the slider from W3, how was translating it to styled components?